apps/files/src/mixins.js

```diff
@@ -45,11 +48,18 @@ export default {
         this.selectedFile = item
         this.newName = item.name
       } else {
-        this.renameFile({
-          client: this.$client,
-          file: this.selectedFile,
-          newValue: item
-        })
+        if (!item.includes('/')) {
+          this.renameFile({
+            client: this.$client,
+            file: this.selectedFile,
+            newValue: item
+          })
+        } else {
+          this.showNotification({
+            title: this.$gettext(`Rename of ${this.selectedFile.name} failed - can not contain "/"`),
+            type: 'error'
```

Comment:
Unrelated to this change: type error should display the notification in red .... something is broken ....

Reply:
I looked at it and the problem is in the name of type prop - should be status. I guess it's still after vuetify maybe and after creating ods component it wasn't changed.